spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
+  before :each do
+    user = create :user, admin: true
```

Comment:
Right, because [the spec helper method](https://github.com/umts/umasstransit.rodeo/commit/aed0c165cf92b57b8ed74c4948a12348fe09aaab#diff-93830fa29d616f7c87903d08b5b1b29aR36) uses `login_as`. Hence my question:

> What's the difference between `sign_in` and `login_as`?